donate/templates/pages/core/campaign_page.html

```diff
@@ -2,9 +2,32 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block content %}
-    <h1>{{ page.title }}</h1>
-    {% image page.hero_image original %}
-    <p>{{ page.lead_text }}</p>
+    {% image page.hero_image width-1200 as image %}
 
-    {% include "fragments/donate_form.html" %}
+    <div class="layout">
+        <div class="layout__container">
+            <div class="layout__full">
+                <div class="hero" {% if image %}style="background-image: url('{{ image.url }}')"{% endif %}>
+                    <div class="hero__container">
+                        <div class="hero__content">
+                            <h1 class="heading heading--primary hero__heading">{{ page.title }}</h1>
+                            <p class="hero__description">{{ page.lead_text }}</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="layout__container">
+            <div class="layout__secondary-col">
+                Content TBC
+            </div>
+            <div class="layout__primary-col">
+                {% include "fragments/donate-form.html" %}
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block extra_js %}
```

Comment:
This block needs to be removed - it is provided by the landing page that this template now extends.